src/main/java/de/letsdev/maven/plugins/ios/ProjectBuilder.java

```diff
@@ -267,26 +270,34 @@ private static void generateIpaArchive(Map<String, String> properties, MavenProj
     }
 
     private static void zipFrameworkArchive(Map<String, String> properties, List<String> targetDependencies,
-                                            String frameworkName, File targetWorkDirectory) throws IOSException {
+                                            String frameworkName, File targetWorkDirectory,
+                                            File projectDirectory) throws IOSException {
         // Zip Frameworks
         String targetZipPath = "../" + properties.get(Utils.PLUGIN_PROPERTIES.APP_NAME.toString()) + "."
                 + Utils.PLUGIN_SUFFIX.FRAMEWORK_ZIP.toString();
         List<String> zipCommandParams = new ArrayList<>();
         zipCommandParams.add("zip");
         zipCommandParams.add("-r");
         zipCommandParams.add(targetZipPath);
-        zipCommandParams.add(frameworkName);
+        zipCommandParams.add(projectDirectory.toString() + "/" + frameworkName);
         zipCommandParams.addAll(targetDependencies);
 
         ProcessBuilder processBuilder = new ProcessBuilder(zipCommandParams);
         processBuilder.directory(targetWorkDirectory);
         CommandHelper.performCommand(processBuilder);
     }
 
-    private static void generateXcFramework(List<String> frameworkPaths, String frameworkTargetName,
-                                            File targetWorkDirectory) throws IOSException {
+    private static void generateXcFramework(Map<String, String> properties, List<String> frameworkPaths,
+                                            String frameworkTargetName, File targetWorkDirectory,
+                                            File workDirectory) throws IOSException {
 
         StringBuilder buildCommand = new StringBuilder();
+
+        if (properties.containsKey(Utils.PLUGIN_PROPERTIES.XCODE_BUILD_COMMAND_WRAPPER_EXECUTABLE.toString())) {
```

Comment:
das containsKey ist immer true oder? Du musst wahrscheinlich eher auf null und empty() prüfen

Reply:
wenn der Key nicht in der Pom angegeben ist, ist das false